example/src/screens/Room.tsx

```diff
@@ -29,7 +29,7 @@ export const Room = ({ navigation }: Props) => {
 
   const { isDevMode, roomName, disconnect } = useVideoroomState();
   const { selectedAudioOutputDevice } = Membrane.useAudioSettings();
-  const participants = Membrane.useRoomParticipants();
+  const participants = Membrane.useEndpoints();
```

Comment:
In the app we can refer to endpoints as "participants" since in the context of the app it makes sense - the endpoints are users that join, talk, have a camera etc. In the context of a library the endpoints are just "network endpoints" have tracks - it can be a security camera that is streaming something, or an user or something else. At least that's how I understand this and that's why we're refactoring it.
And we'll avoid a looot of refactoring :P